Misc/NEWS.d/next/Library/2024-12-20-18-54-53.gh-issue-128136.tgbvWt.rst

```diff
@@ -0,0 +1,8 @@
+New parameter added to the write function that allows you to customize the xml declaration, which was previously hard-coded.

```

Comment:
As I mentioned, the NEWS entry must be shorter (one sentence in general).
Examples should be put in the documentation instead. In addition, a What's New entry must be created for new features.